Week1/homework/app.js

```diff
@@ -1,11 +1,165 @@
KAODUYelTqvMQgbEQF7QBOTQ4chPj0fwbN9cqguHEHLixzUGIEYeoqvLCPH2gm0si3LQNTEDKEi2NbRQmotbNmmnUf4t6OQMk8s0deK9wQ+U9CGVd2nf/dklAI+Efy33gwgSVGrXEjbVSkkGgfM1XDsJIacI9Z67KMe7KHVQsoONeIJKySe9BUKkXts9mtyk6IIAmV0QJemAA74A36ZhrFCkrdmp76xp/XAVnkrQFgJIUdVoSfWgQNAukyt0l68JTHg4CSzQqXY3su4UbiB3/vA+6AHzc9xVZbI7t+C1FP1rMNc0lzTlyiBBD0Sv1mAX1ryIovXXjQTT+/DI+wvdyvhFKU2lK/DF46Tysye7rzG47DAr+jIJtmdgSra94GpplGzjjBaoQnpYSkyutgogV0zKO3qp0nBhqyybQnTuJmXUPnXyMbINtT7mBm7SRfUXBfwZX3pfdxTE0k8J+A04ZYC9zgnPAY4YlvqvTwueDGfu1gcYGIJvfEMGkSJOaYCadA8hFb6f8biHV59Q3n8QV30SjnI/OsH09TRxszc5DhfZ8NJJLINg76d5VroHT0fmB6xcnLeC1BaB6vtSS+Brmde/n12Sxz4QvFmUKQoeX2Q0TJQ7KgqlcNcHf3iFSm25MZSN/X2nyiktuc7jwOehudUn83wLPwIP+Bqs5TmRVb4b8L8aTWRTqL3c0bd9QpkWRJ48gEMSWg1fw24jUzDgxl1g2lnJc39+iyoup/GSjiw2cQIvRc5OA20yew6t3mcVyesmNHPNqt4yxlqbaQH0lV6kyd1VQuIIDio9RwGgtc4L3ya61ucC9tehkGw4/qLdn++NG1xBDPMnuH5bVS38AYveiz5IKNUcj/MbIrQHiRABLkahokugc2n7yS9YjlpqgSWoBhZyvQ2s9vUpNdi0UBl7KyPCgxksE1jZ2/fgSU8Ha6ryGkvubIW7QiIJ/I2zK8w5kR9S+vglKdqMQro44fYqt/VQSUq1gD3wwEy7Ic/GrsapVlhxcCdgC9EELtDDuznzIwTqLJjmbCgpbAln2vL/C7HyAc2wJavzz/mmg3VHwYHUvtyOsRcVpG7itWEYeXYuQEJnzfzWBRMXoECudWPJgi6vYqJelugQQsVtaAb9UqDuXBgqmrE9O59iEI4QqcBhrZ2cqUFkKGG+b19wHF76/0PxkyzJPYJRJFQqLS3G06SAmuDTSlmVv90I3mHIczDOtOEtv0rBkCTT8LvwTMZGpxwoKxRuOQBF7eALvAUFXSVAk8BM1eKLzIKKMYcTjnCUtoSSw4Uvmo0S9uAMJHRzKynbqq8PkYQuSCkUrNT8Fqn+8Y3vshm7xScG1wFzsqYogUPuZLmdHiJaqTnXxeEAkvS4/uivqreJBrBmWjDWI3jKsBzEpDo6lgjVQckFwPOoylBMdf3v29s2jDvc9l950eHlpCnaSToj3gTbqVcn3f6W8AgciTSZFLKhxdJM/h6w0L27iAzvj2aUodOl//PcjpCLceTCi5t8F7CJ3gWYG8t396E9AQ+WNqvYlJVzpbTO2jX3VxH8e2FhRlltuC2uuc1BZ8LxkjoGgPusx0YbR7NscPuouw9KxS0ekHw2XkK29xzaYwVAihO/iuItw3p9H7iPH4uVzgWUKCmmLKzLrw3AXXkxyxyu+vFiXci5LRGeVy8/IyDiiotFbmyZ3aRxkzfxtBS6nVenb7f3/d8N4Nf/FpUgcN3FB76chy3ZIAIYgOIPlllO2FyNv7YcuTATg1oOi9OBA5HYK344rD/s53/mC9uo2ylbwC9RjV6y3o1eI3JAPAzk3b8Pyh25Y01C9pfKjBs/1HXJ+OggrOCLxqZJdT1utQEVilEZOj70BnvfpUP7LH+T7jVuE0J96ZGtg48sixOKIEzD7j4Z82sDdgagLfPox0Bai6CFFIw0bzb3O1c997bUVC24DqfFFQ6xnxL2dt1zLo7CIs4z1ogJ/Kgk4m8uRfc+uuqdd10uhe/k3532NSj9g8jEl055xOemXlMsSbF7ApncW1OOrWxczxFAKUKN/xMtftbSTuWLmH7jiWte3VAADCLD8+Pk/b6Cs4uPIxvNDByHNRhR7P+EwuTPRpI1uMjuoTDL8F9v95GbefVZnJJpBmbgpy5wyUu7UZyK9xEQE2Zqp/sTYl8rC7RNTV413INljb7SBSA3ex2CCCw2ncNWOQV5ZS0btW2FzTQA3MOaiOYwTKlQ/Umyl+QypEIXVucHQy8MRcKbPuD4wbFYB3mXTuEOSVIMdOCiAdSozFvboVSpi95p00bU6skZb2XFdpTH5F7w9e1dYwk5tKlsN7+pljUk5VM2h5zeENAu6CgJwKfY4muk6+Er3ILDb40myql5+sT6oBxPmSM/63Bd9d32HL9MptE4ij8HqtyeqHSmksPn87uL2j6cV1TxlXjRofE10c9WAVnRrTAl16LTBha7jIMBnzXdrvtTmyminT+qVRRHlSGubDvE7qYkngO9e/dOiYYc+Jai9kh3c1XcD3BwjvY4NC54OeFP5yX726pIpRxG+FbV/GlaHc2zGJuYfCI159xu71Fz98beq6bDVXio7J3i50hFHkAJLqOkEmLY2YH2aS1kRx4f6ADPcOuQgJ3FCDCtNkGzMUnw8ZlBE2jcNjix+/DInlCNmAEpClmNhS/0GwsaapVy/3U1dE4xeI0SAPKqaAFY7/bkctt4W5VUddv3KsvBsDpU80raJEmyuZ4LcmtHKJt42zUebWOobCcERtAR0aHt6tNcj2+fJdA4LJ937hMTuCqYdQIGJwFu3FizuQbebu4KR2tWLy7jJ8AkIiM/hllpn28FI02uTGu6GH0F8wqiSqmGR+EC4WyYV9frm/TB69qYgY/36ndC1X/gkdmHc4pop0K5uObBZJKe9ErQIKCM4isTDwARQYEn9DEOBHJCJ9yBkx0AZrSmAA9A+adcgANACuIpRyyRSVuqoBHapWmuFvhHgloMZPL4vOfcZeF5CBlMjrlCoAAAA=',
```

Comment:
I would not put large images as base64 into js like this. It makes the js file very big. Only do this with small images like icons. Store larger images as separate files.